code/modules/projectiles/projectile.dm

```diff
@@ -15,30 +16,54 @@
 	generic_canpass = FALSE
 	blocks_emissive = EMISSIVE_BLOCK_GENERIC
 	layer = MOB_LAYER
-	//The sound this plays on impact.
+	/// The sound this plays on impact.
 	var/hitsound = 'sound/items/weapons/pierce.ogg'
-	var/hitsound_wall = ""
+	/// Sound played when the projectile hits a wall
+	var/hitsound_wall
 
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
-	var/def_zone = "" //Aiming at
-	var/atom/movable/firer = null//Who shot it
-	var/datum/fired_from = null // the thing that the projectile was fired from (gun, turret, spell)
-	var/suppressed = FALSE //Attack message
-	var/yo = null
+	/// Zone at which the projectile was aimed at
+	var/def_zone = ""
```

Comment:
Nitpicking your choice of words ...
```suggestion
	/// Zone at which the projectile is aimed at
	var/def_zone = ""
```
...Because the def zone changes in impact